apps/connection/src/connection.service.ts

```diff
@@ -113,7 +113,7 @@ export class ConnectionService {
    * @param user
    * @returns Connection legacy invitation URL
    */
-  async getConnectionWebhook(payload: IConnectionInterface): Promise<object> {
+  async getConnectionWebhook(payload: Connection): Promise<object> {
```

Comment:
Replace object as a return type with appropriate interface.

Reply:
It's not clear what data will come from the webhook, so its return type has been kept as an object. This was decided after a discussion with Ankita Ma'am and you, that for the webhook's APIs, the return type should remain as an object.